### ISSUE TITLE
FeatureContext support

### ISSUE THREAD

**Issue by drothmaler** [kevholditch/StronglyTypedContext#1]
It would be nice to have support for Typed FeatureContext's too.

**Comment by kevholditch** [comment]
Out of interest are you actively using the StronglyTypedContext?

**Comment by drothmaler** [comment]
@kevholditch yes, I'm currently participating in a huge project, where we use SpecFlow to document all the requirements and for our integration tests. And we are using a strongly typed ScenarioContext in most of the features.
